.github/workflows/build.yml

```diff
@@ -41,7 +41,8 @@ jobs:
           pip install numpy
           cmake -DCMAKE_INSTALL_PREFIX=%cd:\=/%/install -DNLOPT_TESTS=ON .
           cmake --build . --config Release --target install
-          xcopy %cd%\install\bin\nlopt.dll %cd%\src\swig
-          xcopy %cd%\install\Lib\site-packages\_nlopt*.pyd %cd%\src\swig
+          xcopy %cd%\install\bin\nlopt.dll %cd%\src\swig\python
+          xcopy %cd%\install\Lib\site-packages\_nlopt*.pyd %cd%\src\swig\python
+          xcopy %cd%\install\bin\nlopt.dll %cd%\src\swig\Release
```

Comment:
looks like the right path
D:\a\nlopt\nlopt\src\swig\Release\nloptjni.dll
maybe another java dll is missing ?

Reply:
Unfortunately, this part (which is probably the last error making this fail in the CI) is still unclear to me. `nloptjni.dll` should not have any dependencies other than `nlopt.dll`. JNI libraries normally do not link to any Java library. In any case, when I build NLopt with cross-MinGW, it does not have any extraneous dependency. (Depending on the build flags, there can be dependencies on MinGW DLLs, but this should not be the issue for the MSVC build either.)

It sounds more to me like `nlopt.dll` is not being found even if it is in the same directory as `nloptjni.dll`.